README.md

```diff
@@ -1,44 +1,50 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# :construction: IAPPS Web App Front-End
 
-## Available Scripts
+This repo contains the setup for IAPPS Frontend.
 
-In the project directory, you can run:
+## DO NOT ADD ANY EXTERNAL LIBRARIES
 
-### `npm start`
+Everything needed to develop the project has been setup properly including routing, css libraries, liniting files, build process. DO NOT add any external library without discussion. Please simply take a pull from develop and start adding HTML, CSS and TypeScript Code.
 
-Runs the app in the development mode.<br>
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+## Collaboration Guidelines :checkered_flag:
 
-The page will reload if you make edits.<br>
-You will also see any lint errors in the console.
+### Code Quality :art:
 
-### `npm test`
+* No class based Components. Use [hooks](https://reactjs.org/docs/hooks-overview.html#state-hook) only.
+* No implicit types. To ensure the code is of the highest quality, use explicit types when declaring interfaces.
+* Always develop the individual component in isolation using [React Storybook](https://storybook.js.org/). Do not add components directly to app. Check for all possible inputs and scenarios in storybook first.
+* Always use [code splitting](https://facebook.github.io/create-react-app/docs/code-splitting) using ```React.lazy``` and ```React.suspense```
+* Make sure to use [error boundaries](https://reactjs.org/docs/error-boundaries.html#introducing-error-boundaries)
+* Justify every single line of code you write. If you don't know what your code is doing or why did you include it, it will be removed.
+* Use [CSS modules](https://facebook.github.io/create-react-app/docs/adding-a-css-modules-stylesheet) only. No global CSS is allowed. Every component should be responsible for its own presentation.
```

Comment:
If you are using CSS modules then what is the point of including tailwind? Pick one thing and stick with it.

Reply:
Good Point. If someone needs to use tailwind, they can add it manually.